Makefile

```
@@ -34,7 +35,12 @@ build-ui:               ## Build platform-ui
 	lerna run build --scope='@percona/platform-ui'
 
 docker-build:           ## Build Docker image
-	docker build --squash --tag $(DOCKER_IMAGE) .
+	docker build --squash --tag $(DOCKER_IMAGE):$(DOCKER_TAG) .
 
 docker-push:            ## Push Docker image
-	docker push $(DOCKER_IMAGE)
+	docker push $(DOCKER_IMAGE):$(DOCKER_TAG)
+
+docker-tag:
+	if [[ "$(DOCKER_TAG)" == "latest" ]]; then exit 0; fi; \
+	    docker tag $(DOCKER_IMAGE):$(DOCKER_TAG) $(DOCKER_IMAGE):dev-latest; \
+	    docker push $(DOCKER_IMAGE):dev-latest;
```

```suggestion
	if [[ "$(DOCKER_TAG)" != "latest" ]]; then
	    docker tag $(DOCKER_IMAGE):$(DOCKER_TAG) $(DOCKER_IMAGE):dev-latest
	    docker push $(DOCKER_IMAGE):dev-latest
	fi
```
This reads better and is more straightforward )

---

It looks like we will prevent the engineers from developing in parallel, since every container in every PR will be pushed with `dev-latest` label, so the last push will overwrite the previous one. Let's discuss one more time...

---

> This reads better and is more straightforward )

Added semicolons and backslashes to make this work :)

```suggestion
	if [[ "$(DOCKER_TAG)" == "latest" ]]; then \
		docker tag $(DOCKER_IMAGE):$(DOCKER_TAG) $(DOCKER_IMAGE):dev-latest; \
		docker push $(DOCKER_IMAGE):dev-latest; \
	fi; \
```

---

I guess it should be `!=` :)

---

Well it's better to be like this, for sure. As for why it's presented like that, because it's a leftover from previous attempt /experiment, was a bit quicker to add the exit :worried:

---

> I guess it should be `!=`, because `== latest` was meant to exit :)

oopsie